SergiD/medium/practice.css

```diff
@@ -0,0 +1,276 @@
+    font-size: 70px;
+    font-weight: 400;
+    color: rgba(8, 8, 8, 1);
+    line-height: 74px;
+    font-family: gt-super, Georgia, 'Times New Roman', Times, serif;
+}
+.welcome_link
+{
+    text-decoration: underline;
+    cursor: pointer;
+}
+.welcome_text
+{
+    width: 600px;
+    height: 70px;
+    font-size: 18px;
+}
+.btn_banner
+{
+    background-color: #FFC017;
+    border: 1px solid black;
+    border-radius: 4px;
+    cursor: pointer;
+}
+.image_banner
+{
+    width: 360px;
+    height: 320px;
+    position: relative;
+    padding-top: 30px;
+    bottom: 30px;
+    right: 30px;
+}
+
+.top_container
+{
+    width: 1192px;
+    height: 300px;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    /* border: 1px solid black; */
+    position: relative; left: 75px;
+}
+.trl_container
+{
+    width: 1192px;
+    height: 50px;
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    font-size: 16px;
+    font-family: Arial, Helvetica, sans-serif;
+    /* border: 1px solid black; */
+
+}
+
+.nw_container
+{
+    width: 1192px;
+    height: 250px;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: flex-start;
+    /* border: 1px solid red; */
+
+}
+
+.container_one
+{
+    width: 30%;
+    height: 120px;  
+    color: olive;
+    display: flex;
+    flex-direction: column;
+    /* border: 1px solid green; */
+}
+
+.container_two
+{
+    width: 30%;
+    height: 120px;
+    color: olive;
+    display: flex;
+    flex-direction: column;
+    /* border: 1px solid green; */
+
+}
+.container_three
+{
+    width: 30%;
+    height: 120px;
+    color: olive;
+    display: flex;
+    flex-direction: column;
+    /* border: 1px solid green; */
+}
+.container_four
+{
+    width: 30%;
+    height: 120px;
+    color: olive;
+    display: flex;
+    flex-direction: column;
+    /* border: 1px solid green; */
+}
+.container_five
+{
+    width: 30%;
+    height: 120px;
+    color: olive;
+    display: flex;
+    flex-direction: column;
+    /* border: 1px solid green; */
+}
+.container_six
+{
+    width: 28%;
+    height: 120px;
+    color: olive;
+    display: flex;
+    flex-direction: column;
+    /* border: 1px solid green; */
+}
+.number_container
+{
+    width: 3%;
+    height: 20px;
+    font-size: 30;
+    color: rgb(163, 232, 232);
+    display: flex;
+    justify-content: center;
+    /* border: 1px solid green; */
+}
+
+.grid_container
+{
+    width: 600px;
+    height: 100px;
+    display: grid;
+    grid-template-columns: repeat(4 ,150px);
+    grid-template-rows: repeat(5 , 20px) 
+}
+/* .grid_container>*
+{
+    border: 1px solid black;
+} */
+.gc_one
+{
+    grid-column-start: 1;
+    grid-column-end: 5;
+}
+/* .gc_two .gc_three .gc_four .gc_five .gc_six .gc_seven .gc_eight .gc_nine .gc_ten
+{
+    display: grid;
+    barder: 1px solid black
+    grid-gap: 10px;
+} */
+.gc_eleven
+{
+    grid-row-start: 5;
+    grid-column-start: 1;
+    grid-column-end: 4;
+
+}
```

Comment:
General notes on CSS:
1 - Delete all comments.
2 - In blocks {} delete null strings.
3 -  Stick to one syntax.